### PR TITLE
Ilich Edits

### PR DESCRIPTION
**Pull request by ailich**
- Add github actions to check CRAN requirements
- Specify Imports
- Add installation instructions
- Have terra::terrain return radians instead of converting
- A few minor code cleaning things

### CODE REVIEW

File: .gitignore
Comment (strevisani) [inline .gitignore:1]: just wondering if putting in the list ".RData" creates problems on the package in loading the kernels for computing directional differences that are saved in this format

File: DESCRIPTION
Comment (strevisani) [inline DESCRIPTION:1]: Happy if you would like to add yourself as contributor, as you suggested:
person("Ilich", "Alexander", role = "ctb", comment = c(ORCID = "0000-0003-1758-8499"))
I added, hopefully correctly...

File: NAMESPACE
Comment (strevisani) [inline NAMESPACE:1]: It is fine to export, even if when I developed these was just for doing the comparison with MAD.

File: R/GeostTextureLibrarySharedRoxy.R [hunk 1]
```diff
@@ -396,7 +395,8 @@ CalcMeans=function(deltas,w,exponent){
 #' @param kernels The kernels to be used for computing the directional differences (e.g. order 1 or 2 for various lags)
 #' @param w The moving window adopted for computing the geostatistical index (i.e., MAD)
 #' @param exponent The exponent: increasing the exponent increase the sensitivity to outliers. Set 2 for Variogram and 1 for Madogram.
-#' @return A list of 3 rasters: 1)isotropic roughness; 2) direction of anisotropy; 3)index of anisotropy.
+#' @return A SpatRaster with 3 layers: 1)isotropic roughness; 2) direction of anisotropy; 3)index of anisotropy.
```
Comment (strevisani) [inline R/GeostTextureLibrarySharedRoxy.R:398]: yes, you are right it is a 3 layers raster not a list!

**Review by strevisani — APPROVED** [review]

File: .gitignore
Comment (ailich) [reply .gitignore:1]: This seems to only ignore a file called ".RData" which is if you save your environment to load into R when closing Rstudio. It doesn't seem to ignore all files with that extension so it should be good as is.

File: R/GeostTextureLibrarySharedRoxy.R (hunk 1 above)
Comment (ailich) [reply R/GeostTextureLibrarySharedRoxy.R:398]: Cool, I saw "list of rasters" mentioned a few times on the input and output side of the documentation so it's probably worth a quick check to change those over to saying "multilayer SpatRaster" in the documentation.

**Pull request merged**